ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -321,4 +321,8 @@ if test "${ret}" -eq 0 ; then
   echo "https://$(env KUBECONFIG=${dir}/auth/kubeconfig oc -n openshift-console get routes console -o=jsonpath='{.spec.host}')" > "${SHARED_DIR}/console.url"
 fi
 
+echo "install-config.yaml"
+echo "-------------------"
+cat ${SHARED_DIR}/install-config.yaml | grep -v "password\|username|pullSecret" | tee {ARTIFACT_DIR}/install-config.yaml
```

Comment:
```
grep -v "password\|username|pullSecret"
```

should be (note missing second backslash):

```
grep -v "password\|username\|pullSecret"
```

According to [4.11 install-config.yaml](https://docs.openshift.com/container-platform/4.11/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#additional-resources_config) those seem to be the only sensitive info.

Reply:
And `{ARTIFACT_DIR}` I believe should be `${ARTIFACT_DIR}`

Reply:
Thank you for catching these mistakes. guess I was cruising to the long weekend too carelessly. :)

just updated with the fixes